src/events.jl

```diff
@@ -135,6 +135,7 @@ function attachWindowSizeCallback(scene, camera)
 	callback = (_, w, h) -> begin
 		camera.aspectRatio *= w/scene.canvas.size[1]
 		camera.aspectRatio *= scene.canvas.size[2]/h
+		camera.fov = 2*atan( (h/(scene.canvas.size[2])) * tan((camera.fov)/2) )
```

Comment:
I was thinking about it yesterday. `fov` is camera intrinsic. Some applications might need constant `fov`. Intuitively it shouldn't change for certain applications like splat.jl may not like it.

Reply:
We can merge this. I will raise an issue/draft to do more research on it.